### PR TITLE
Vp-976 Admin/ OrgAdmin can create new blog posts

### PR DESCRIPTION
**Pull request by farjanacan**
## I do solemnly swear that I have:
- [x] Run `npm test` and all the tests pass.
- [x] Run `npm run lint` and there are no warnings.
- [ ] Not decreased the overall test coverage?
- [x] Included the Jira ID and description in the PR title

## Proposed Changes? 🤔
1. New blogs can be created from the "Updates" Tab in any opdetail page
2. The Blogs display in the same tab when created
3. The author name and time posted is shown in a footer

### Original
Original

### Updated
Updated

### CODE REVIEW

File: server/api/story/story.controller.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
 const Story = require('./story')
 
+/**
+  api/stories/ -> list all the stories assigned to the opportunity and get the story details
+ */
+
+const listStory = async (req, res) => {
+  const parentId = req.query.parentId
+  const query = { parent: parentId }
+  console.log('listStory', req.query, query)
```
Comment (avowkind) [inline server/api/story/story.controller.js]: please remove console.log statements for PRs to master.

File: components/Op/OpTabs.js [hunk 2]
```diff
@@ -46,9 +46,9 @@ const opEditTab =
 
 const isNotProd = process.env.NODE_ENV !== 'production'
 
-export const OpTabs = ({ op, onChange, canManage, defaultTab }) => (
+export const OpTabs = ({ op, onChange, canManage, defaultTab, author }) => (
   <>
-    <VTabs size='large' defaultActiveKey={defaultTab} onChange={onChange}>
+    <Tabs style={shadowStyle} size='large' defaultActiveKey={defaultTab} onChange={onChange}>
```
Comment (avowkind) [inline components/Op/OpTabs.js]: I'm unclear why you are changing the tabs. not required for your story.

File: components/Op/OpTabs.js [hunk 3]
```diff
@@ -71,7 +71,7 @@ export const OpTabs = ({ op, onChange, canManage, defaultTab }) => (
       {canManage && (
         <TabPane tab={opEditTab} key='edit' />
       )}
-    </VTabs>
+    </Tabs>
```
Comment (avowkind) [inline components/Op/OpTabs.js]: Please stick to using VTabs

File: components/Op/OpUpdatePanel.js [hunk 4]
```diff
@@ -1,37 +1,58 @@
...
+  useEffect(() => {
+    const getStories = async () => { await dispatch(reduxApi.actions.stories.get({ parentId: op._id })) }
+    getStories()
+  }, [])
+
+  const setStory = async (story) => {
+    // save back to redux and mongo
+    story.parent = op._id
+    story.author = author
+    console.log('setStory', story, op._id)
```
Comment (avowkind) [inline components/Op/OpUpdatePanel.js]: remove console.log

File: components/Op/OpUpdatePanel.js [hunk 5]
```diff
@@ -1,37 +1,58 @@
...
-Our cognitive strategic staircases interact with a potential shareholder value across the wider Group; nevertheless our ever-changing and/or non-linear digital change operationalizes our nimble, awesome, developer-led and well-scoped correlations. Internally and externally, the visionary commoditizes our top-down expectations.<br /><br />Our sprint-based and extensible responsive design inspires the enabler.
-It's not about win-win solutions. It's about an all-encompassing resiliency. Quality assurance requires that we all pull in the same direction. The prioritizing correlations energize the game changers.
-The board-level executives consistently transgenerate our siloed ingenuity in the marketplace.
-A stress management will be best positioned to re-imagine unmatched, cognitive and cognitive risk appetites, whereas opting out of usage-based leadership strategy is not a choice.
-.
-        </p><br /><Button shape='round' size='large' type='secondary'>Reply</Button>
+        {
+          stories.data.length === 0 ? (
+            <P>There is no update</P>
```
Comment (avowkind) [inline components/Op/OpUpdatePanel.js]: Use formattedMessage and default to 'Check here for updates about this activity'.

File: components/Op/OpTabs.js (hunk 2 above)
Comment (farjanacan) [reply components/Op/OpTabs.js]: That happened due to a merge conflict, my branch was outdated initially. I've changed it back to what it should be.

File: components/Op/OpTabs.js (hunk 3 above)
Comment (farjanacan) [reply components/Op/OpTabs.js]: Done

File: components/Op/OpUpdatePanel.js (hunk 5 above)
Comment (farjanacan) [reply components/Op/OpUpdatePanel.js]: Done

**Pull request merged**